Dockerfile

```diff
@@ -1,11 +1,14 @@
 FROM alpine:edge
 LABEL maintainer="dev@jpillora.com"
+
+ENV DNSMASQ_VERSION 2.82-r0
+
 # webproc release settings
 ENV WEBPROC_VERSION 0.2.2
 ENV WEBPROC_URL https://github.com/jpillora/webproc/releases/download/$WEBPROC_VERSION/webproc_linux_amd64.gz
 # fetch dnsmasq and webproc binary
 RUN apk update \
-	&& apk --no-cache add dnsmasq \
+	&& apk --no-cache add dnsmasq=$DNSMASQ_VERSION \
```

Comment:
For the same reasons that I don't advise `edge` builds beyond local use/testing, I would advise against version pinning, especially on `edge` repo. See [official docs warning](https://wiki.alpinelinux.org/wiki/Alpine_Linux_package_management#Holding_a_specific_package_back):

> It is important to understand your package repo's version retention rules. 
> For example, most Alpine package repos contain an "edge" branch, which may drop package versions that are not deemed fit to make it into a stable branch. 
> This means that pinning to a version on the edge branch may stop working after the package version is revoked from the repo. 
> Always pin to a package version that is intended for your current Alpine Linux version.

Thus, this will likely break in future should anyone try to build this Dockerfile after releasing to Dockerhub. Which has tags that don't match dnsmasq releases either mind you.